tests/route/test_default_route.py

```diff
@@ -107,13 +106,14 @@ def verify_default_route_in_app_db(duthost, tbinfo, af, uplink_ns, device_neigh_
 
 
 
-def test_default_route_set_src(duthosts, tbinfo):
+def test_default_route_set_src(duthosts, enum_rand_one_per_hwsku_frontend_hostname, 
+                               enum_rand_one_asic_index, tbinfo):
     """
     check if ipv4 and ipv6 default src address match Loopback0 address
 
     """
-    duthost = find_duthost_on_role(duthosts, get_upstream_neigh_type(tbinfo['topo']['type']) , tbinfo)
-    asichost = duthost.asic_instance(0 if duthost.is_multi_asic else None)
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
```

Comment:
why this change ? It works fine

Reply:
I see this error:
>       pytest_assert(role_host, "Could not find {} duthost".format(role))
E       Failed: Could not find T3 duthost

because `get_upstream_neigh_type(tbinfo['topo']['type'])` is T3 for chassis

Reply:
T2 should have T3 neighbor in the graph . Why it is failing ?

Reply:
tthen might be minigraph is wrong? let me confirm

Reply:
@abdosi I checked Nokia nightly with this this change works, it fails on arista testbed might due to the wrong minigraph on single-asic card, please review again